grants_tagger/streamlit_visualize.py

```diff
@@ -29,15 +41,40 @@
 
 probabilities = st.sidebar.checkbox("Display probabilities")
 
+if text == DEFAULT_TEXT:
```

Comment:
streamlit tends to run everything in the script so it would go and calculate predictions and explanations for the default text. this is how you tell streamlit to not execute everything in some cases. See https://docs.streamlit.io/en/stable/api.html#control-flow